src/test/java/io/neonbee/NeonBeeMockHelper.java

```diff
@@ -187,7 +191,18 @@ public static Future<NeonBee> createNeonBee(Vertx vertx) {
      * @return the mocked NeonBee instance
      */
     public static Future<NeonBee> createNeonBee(Vertx vertx, NeonBeeOptions options) {
-        return NeonBee.create((vertxOptions) -> succeededFuture(vertx), options, null);
+        return NeonBee.create((vertxOptions) -> {
+            if (options.isClustered() && vertxOptions.getClusterManager() != null && vertx instanceof VertxImpl) {
+                try {
+                    ClusterManager cm = vertxOptions.getClusterManager();
+                    cm.init(vertx, null);
+                    ReflectionHelper.setValueOfPrivateField(vertx, "clusterManager", cm);
```

Comment:
I have my concerns with this code. Since the clusterManager is used to make various decisions during the initialization of Vertx and we set a clusterManager here afterwards, I could imagine that we cause consequential errors here.

Reply:
This hack was only required for one test, but you are right! This is not very clean and fits only to this one specific test case. I re-wrote this test in a way that this hack is not required.